src/test/scala/com/redislabs/provider/redis/rdd/RedisStandaloneEnv.scala

```diff
@@ -6,17 +6,14 @@ import org.apache.spark.SparkConf
 /**
   * @author The Viet Nguyen
   */
-trait RedisStandaloneSuite extends SparkRedisSuite {
-
-  private val redisHost = "127.0.0.1"
-  private val redisPort = 6379
-  private val redisAuth = "passwd"
+trait RedisStandaloneEnv extends ENV {
```

Comment:
I would also move `Env`, `RedisClusterEnv` and `RedisStandaloneEnv` from `rdd` to a separate package `env`, so that `rdd` package only contains tests. It would be also good to move `RedisConfigSuite` from `rdd` package since it doesn't actually test RDD.